org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/XMLLanguageClientAPI.java

```diff
@@ -32,4 +36,10 @@
 	default void actionableNotification(ActionableNotification command) {
 		throw new UnsupportedOperationException();
 	}
+	
+	@JsonRequest("executeClientCommand")
```

Comment:
remove this JSonRequest to be consistent with actionableNotification (or add JsonRequest for actionableNotification).

Reply:
Add javadoc

Reply:
I notice that JDT LS provides this kind of method in https://github.com/eclipse/eclipse.jdt.ls/blob/cf6358ba4c166fa7d4c45aba5b7b22ea08516524/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/lsp/ExecuteCommandProposedClient.java

```java
@JsonRequest("workspace/executeClientCommand")
CompletableFuture<Object> executeClientCommand(ExecuteCommandParams params);
```

I wonder if we could use the same "workspace/executeClientCommand" but I'm afraid that it will interference with vscode-java. @fbricon what do you think about that?

Reply:
Fixed, but keeping not resolved to see if we are switching to `workspace/executeClientCommand` - think it'd be good and won't affect the JDT LS clients

Reply:
Keep xml/executeClientCommand to avoid conflicts

Reply:
Ok thanks @fbricon let's see like this.